jenkins-x.yml

```diff
@@ -0,0 +1,61 @@
+pipelineConfig:
+  env:
+  - name: GIT_AUTHOR_EMAIL
+    value: jenkins-x@googlegroups.com
+  - name: GIT_AUTHOR_NAME
+    value: jenkins-x-bot
+  - name: GOPROXY
+    value: http://jenkins-x-athens-proxy
+  - name: GOOGLE_APPLICATION_CREDENTIALS
+    value: /usr/local/lib/bazel/etc/google-creds.json
+  - name: PROW_REPO_OVERRIDE
+    value: gcr.io/jenkinsxio/prow
+  pipelines:
+    pullRequest:
+      pipeline:
```

Comment:
It'd probably be good to add `containerOptions` with cpu/memory request/limits like https://github.com/jenkins-x/jx/blob/master/jenkins-x-integration.yml#L27-L35 to the `pipeline`s. Obviously the requests/limits can be higher, but I've found that when we don't specify anything, things can go...poorly.

Reply:
Done.